src/lib/pages/library/ui/library-page.svelte

```diff
@@ -4,9 +4,9 @@
   import { EpisodeCard } from '$lib/widgets/episode';
   import { UserContainer } from '$lib/entities/user';
   import { SignInButton, SignOutButton } from '$lib/features/authenticate';
-  import type { Episode } from '$lib/shared/api';
+  import { getEpisode } from '../lib/get-episode';
 
-  $: episodes = Array.from<Episode>($likesStore.values());
+  const liked = [...$likesStore];
```

Comment:
issue: this will not be updated as the `likesStore` updates

suggestion: use a reactive declaration instead:
```suggestion
  $: liked = [...$likesStore];
```

Reply:
Actually this was intented. Like I said it during my demo, I want te episodes to still be on UI until reload, in case user changes their mind.

Reply:
oh, alright then, I take it back